screens/Home.js

```diff
@@ -19,29 +19,134 @@ import {
   Menu,
   MenuOptions,
   MenuOption,
-  MenuTrigger, renderers
+  MenuTrigger,
+  renderers
 } from "react-native-popup-menu";
 import getStyleSheet from "../styles/themestyles";
 import { ScreenStyles, FontStyles } from "../styles/global";
 import firebase from "react-native-firebase";
 import DatabaseManager from "../components/DatabaseManager";
 import { showMessage } from "react-native-flash-message";
+import AppIntroSlider from "react-native-app-intro-slider";
+
+const tutorialSlides = [
+  {
+    key: "slide1",
+    title: "Welcome to FitnessGo!",
+    text:
+      "This app allows you to create timers for your workouts, discover workouts from other people, and share your own workouts with your friends.",
+    image: require("../assets/images/main/Stopwatch.png")
+  },
+  {
+    key: "slide2",
+    title: "Create workouts",
+    text: "Press plus button to create a workout",
+    image: require("../assets/images/tutorial/1.png")
+  },
+  {
+    key: "slide3",
+    title: "Add your exercises",
+    text:
+      'Type in workout name, select a workout category, and add exercise by selecting an exercise name and typing number of repetitions, duration and break. When done, simply press "Save" button on top right corner.',
```

Comment:
Type in a workout name, select a workout category, and add an exercise by selecting an exercise name. Create sets by entering a duration, break, and number of repetitions.